packages/discord.js/src/structures/GuildAuditLogsEntry.js

```diff
@@ -102,8 +102,8 @@ class GuildAuditLogsEntry {
      */
     this.executor = data.user_id
       ? guild.client.options.partials.includes(Partials.User)
-        ? guild.client.users._add({ id: data.user_id })
-        : guild.client.users.cache.get(data.user_id)
+        ? guild.client.users._obtain({ id: data.user_id }, guild)
+        : logs.users.get(data.user_id)
```

Comment:
What’s logs meant to be? I don’t see it declared anywhere

Reply:
Its the first parameter in the constructor

Reply:
Ah I see
But then what’s the type? Is users.get a function?

Reply:
https://github.com/ckohen/discord.js/blob/aa2bf30a975b47aeeb024b58ce48d130af3cbc9c/packages/discord.js/src/structures/GuildAuditLogs.js#L81